src/Stryker.CLI/Stryker.CLI/StrykerCLI.cs

```diff
@@ -112,8 +113,14 @@ private void RunStryker(StrykerOptions options)
                    this.HandleBreakingThresholdScore(options, results);
                }
             }
-            catch (Exception)
+            catch(StrykerInputException strEx)
             {
+                Console.WriteLine(strEx.Message);
+                this.ExitCode = 1;
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"Something went wrong: {ex.Message}");
```

Comment:
Not sure if I understood correctly your comment.
Yes. If we want only hide stack trace for custom exceptions (`StrykerInputException`) than this is not needed. But we need to set exit code to 1 in case of general `Exception`.
By the way, I think I will move `try` `catch` block one level up. To `Program.cs`.